### ISSUE TITLE
Opinions on using <picture> element

### ISSUE THREAD

**Issue by jordanwillis** [vesdim2007/Colmar#3]
Nice job spending the time to learn about the more semantic `<picture>` element and incorporating this in your project.  I wanted to share with my some thoughts regarding the `<picture>` element in general so that you can make an informed decision if you choose to use it going forward on future projects.

There is a general debate in the industry whether the `<picture>` specification is really such a good thing or good practice to use.  On the one hand, you can use the element to define rules for when different images are displayed and it degrades well for browsers that are not compatible with the element.  On the other hand, the browser must follow the rules setup in the element regardless of how a user might have their browser configured.

On the other hand, you can achieve a similar behavior by using the `srcset` or `sizes` attribute of the regular `img` element.  The benefit of this approach is that it provides more information to the browser for what the images are and allows the browser to decide when to apply the rules based upon how a user might have their browser configured.

Aside from this, I would argue that its better to maintain the separation of concerns between document structure (HTML) and document style (CSS) and to handle responsive images via media queries.  It is always better to adhere to separation of concerns because it simplifies the system and makes it more maintainable.

Here is an article that discussing some of this in more detail if you want to take a look.

https://cloudfour.com/thinks/dont-use-picture-most-of-the-time/

Long story short, there is nothing wrong with your approach and the fact that you learned about the element and used it is great!  I just wanted to share a different perspective so that you can decide how you might use it going forward.
